src/constants.js

```diff
@@ -28,3 +28,5 @@ export const statusFilter = [
   { label: 'ui-users.filters.status.active', value: 'active' },
   { label: 'ui-users.filters.status.inactive', value: 'inactive' }
 ];
+
+export const feeFineBalanceId = 'cf7a0d5f-a327-4ca1-aa9e-dc55ec006b8a';
```

Comment:
With current id I determine that this is fee/fine condition, because this condition is validating with different message, and condition fields are rendering dinamically. All conditions (thare are 6 of them) are always present on BE with hardcoded ids for now.